src/documents/activities/FunctionalMashup.html.md.handlebars

```diff
@@ -7,24 +7,32 @@ category: Activities
 This activity can be used in a co-design or other group design session to encourage participants to work together toward more creative and resourceful solutions, by thinking beyond the current or obvious.
 ## How
 ### Preparation
-You will need a deck of object cards. You can create your deck using any available materials. You might print images of different objects on paper or cardstock, use language learning flashcards, cut out magazine images and paste them onto cards, or find other ways to create a collection of random objects. 
+You will need a deck of object cards. You can create your deck using any available materials. You might print images of different objects on paper or cardstock, use language learning flashcards, cut out magazine images and paste them onto cards, or find other ways to create a collection of random objects.
+<figure>
+[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted on.](/images/object_cards_thumb.jpg)](/images/object_cards_thumb.jpg)
```

Comment:
The thumbnail is missing. Make sure to commit it.

Reply:
thanks! I removed the wrong file...will correct it.